lua/zk/util.lua

```diff
@@ -51,22 +51,45 @@ function M.get_lsp_location_from_selection()
   }
 end
 
+---Fix to correct cursor location
```

Comment:
The nvim cursor is (1,0)-indexed, so lines start at 1. Might be what's causing the issue you noticed.